python/src/opendp/typing.py

```diff
@@ -125,8 +125,8 @@ def __init__(self, origin, args=None):
     def __eq__(self, other):
         if isinstance(other, str):
```

Comment:
It's not a change that is coming in with this PR, but it feels weird that the result of `==` could vary, depending on whether a string is on the LHS or RHS. I feel a bit uncomfortable with the flexibility on RuntimeType arguments, but this PR isn't the place to try to make changes.

Reply:
I mentioned in another comment how parsed primitive/atomic types are still strings, which is why you get this behavior. 
To recap, the result of == should only vary if the LHS is not parsed: like if LHS is "Vec<u64>". The RHS gets promoted, but the LHS does not.

We could revisit this. The equivalent parser in the R bindings wraps primitive types (it made things much easier in that setting).